### PR TITLE
Draw lines between stops and change tile style

### PR DESCRIPTION
**Pull request by emilhaugberg**
There are quite a lot of changes in here, sorry about that.

### CODE REVIEW

**Comment by emilhaugberg** [comment]
Example

Screenshot 2019-06-26 at 15 48 59

**Comment by emilhaugberg** [comment]
Im going to try to add a legend.

**Comment by emilhaugberg** [comment]
Screenshot 2019-06-26 at 16 34 16

File: src/components/Maps/Maps.js [hunk 1]
```diff
@@ -103,6 +133,41 @@ class Maps extends React.Component {
...
+
+    createLines() {
+        return Object.keys(this.state.lines).map(line => {
+            let lineObj = this.state.lines[line]
+            return <Polyline key={line} color={lineObj.color} positions={lineObj.coords} />
+        })
+    }
+
+    createLineLegend() {
+        console.log("comes here")
```
Comment (emilhaugberg) [inline src/components/Maps/Maps.js]: gonna remove this

File: src/components/Grid/Grid.css [hunk 2]
```diff
@@ -250,4 +250,42 @@
...
+    background-color: white;
+    border-radius: 5px;
+    -webkit-box-shadow: 2px 3px 5px 0px rgba(199,199,199,1);
+    -moz-box-shadow: 2px 3px 5px 0px rgba(199,199,199,1);
+    box-shadow: 2px 3px 5px 0px rgba(199,199,199,1);
+}
+
+.line-legend-row {
+    /* width: 90%; */
+    /* display: block; */
```
Comment (emilhaugberg) [inline src/components/Grid/Grid.css]: Forgot to remove this

File: src/components/Maps/Maps.js [hunk 3]
```diff
@@ -125,8 +191,8 @@ class Maps extends React.Component {
                         intensityExtractor={m => parseFloat(m[2])}
                     />
                     <TileLayer
-                        attribution='&copy; Overzicht'
```
Comment (man-netcat) [inline src/components/Maps/Maps.js:128]: I would keep this

Comment (emilhaugberg) [reply src/components/Maps/Maps.js:128]: I think we have to credit the creators of the tile.

**Review by man-netcat — CHANGES_REQUESTED** [review]
One minor change requested, the rest looks great

File: src/components/Maps/Maps.js (hunk 3 above)
Comment (man-netcat) [reply src/components/Maps/Maps.js:128]: Ah, in that case I approve the merge request.

**Review by man-netcat — APPROVED** [review]

**Pull request merged**